repo2docker/docker.py

```diff
@@ -73,6 +73,7 @@ def build(
         dockerfile="",
         fileobj=None,
         path="",
+        labels={},
```

Comment:
Good to avoid mutables in signatures:

```suggestion
        labels=None,
```